src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint_legacy.py

```diff
@@ -432,6 +432,7 @@ def __call__(
         guidance_scale: Optional[float] = 7.5,
         negative_prompt: Optional[Union[str, List[str]]] = None,
         num_images_per_prompt: Optional[int] = 1,
+        add_predicted_noise: Optional[bool] = True,
```

Comment:
Could we set the default to `False` as otherwise it'd be backwards breaking? Note that we don't really have a reference implementation for the "legacy" inpaint pipeline, so it's quite difficult to objectively state what is better. 
```suggestion
        add_predicted_noise: Optional[bool] = False,
```